dashboard/src/app/workspaces/workspace-details/workspace-overview/workspace-details-overview.controller.ts

```diff
@@ -267,12 +267,12 @@ export class WorkspaceDetailsOverviewController {
    */
   onEphemeralModeChange(): void {
     if (this.isEphemeralMode) {
-      this.workspaceDetails.config.attributes.persistVolumes = 'false';
+      this.workspaceDetails.config.attributes['persistVolumes'] = 'false';
     } else {
-      if (this.attributesCopy.persistVolumes) {
-        this.workspaceDetails.config.attributes.persistVolumes = 'true';
+      if (this.attributesCopy['persistVolumes']) {
+        this.workspaceDetails.config.attributes['persistVolumes'] = 'true';
       } else {
-        delete this.workspaceDetails.config.attributes.persistVolumes;
+        delete this.workspaceDetails.config.attributes['persistVolumes'];
       }
```

Comment:
You are using  ```['persistVolumes']``` five times only on this page.
As for me, I prefer to add one more line in  'che.d.ts'

this line 
 ``` attributes?: {[attrName: string]: string}; ```

change to this
```
    attributes?: {
      persistVolumes: string;
      [attrName: string]: string;
    };
```
WDYT?

Reply:
That does makes sense, but when doing that it would maybe be nice to add the other possible configuration attributes also to this interface. These are `editor`, `plugins` and `persistVolumes`.